### PR TITLE
[RELEASE] 0.15.45

### PR DESCRIPTION
**Pull request by Kilo59**
release prep for 0.15.45

- [x] fix links
- [x] remove gx devs
- [x] update teams yaml

### CODE REVIEW

File: docs/changelog.md [hunk 1]
```diff
@@ -2,6 +2,79 @@
...
+* [DOCS] enable running `invoke docstrings` on select modules ([#6868](GxURL.PULL_REQUESTS/6868))
+* [DOCS] Adds docstring for expect_column_distinct_values_to_contain_set ([#6855](GxURL.PULL_REQUESTS/6855))
+* [DOCS] Documentation Strings for Metric Domain Types and Metric Function Types ([#6872](GxURL.PULL_REQUESTS/6872))
+* [DOCS] added docstrings for the public API ([#6884](GxURL.PULL_REQUESTS/6884)) (thanks @sujensen)
+* [DOCS] Add public API docstring for `expect_column_values_to_be_unique` `validate_configuration` ([#6897](GxURL.PULL_REQUESTS/6897))
+* [DOCS] Miscellaneous docstrings for `DataContext` and utils ([#6852](GxURL.PULL_REQUESTS/6852))
+* [DOCS] Add public API docstring for `expect_column_values_to_be_of_type` `validate_configuration` ([#6896](GxURL.PULL_REQUESTS/6896))
+* [DOCS] Add public API docstring for `JsonSchemaProflier.validate` ([#6900](GxURL.PULL_REQUESTS/6900))
+* [DOCS] Exclude DataAssistantRunner.run() ([#6919](GxURL.PULL_REQUESTS/6919))
+* [DOCS] StoreValidationResultAction, StoreEvaluationParametersAction and StoreMetricsAction api docs ([#6879](GxURL.PULL_REQUESTS/6879)) (thanks @josectobar)
```
Comment (Kilo59) [inline docs/changelog.md]: remove?
```suggestion
* [DOCS] StoreValidationResultAction, StoreEvaluationParametersAction and StoreMetricsAction api docs ([#6879](GxURL.PULL_REQUESTS/6879))
```

File: docs/changelog.md [hunk 2]
```diff
@@ -2,6 +2,79 @@
...
+* [MAINTENANCE] Use docusaurus style code block in api docs ([#6906](GxURL.PULL_REQUESTS/6906))
+* [MAINTENANCE] metrics linting ([#6889](GxURL.PULL_REQUESTS/6889))
+* [MAINTENANCE] Add exception message to `RenderedAtomicContent` failure renderer ([#6795](GxURL.PULL_REQUESTS/6795))
+* [MAINTENANCE] Remove CloudNotificationAction ([#6881](GxURL.PULL_REQUESTS/6881))
+* [MAINTENANCE] Use ruff linter for docstring linting ([#6913](GxURL.PULL_REQUESTS/6913))
+* [MAINTENANCE] Add validate_configuration method docstrings ([#6899](GxURL.PULL_REQUESTS/6899)) (thanks @elenajdanova)
+* [MAINTENANCE] docstring for expect_column_values_to_not_match_like_pattern_list's validate_configuration ([#6874](GxURL.PULL_REQUESTS/6874))
+* [MAINTENANCE] docstring for expect_column_values_to_not_match_like_pattern validate_configuration ([#6876](GxURL.PULL_REQUESTS/6876))
+* [MAINTENANCE] docstring for expect_compound_columns_to_be_unique validate_configuration ([#6878](GxURL.PULL_REQUESTS/6878))
+* [MAINTENANCE] Add docstrings for Validator and its save_expectation_suite and validate methods ([#6885](GxURL.PULL_REQUESTS/6885)) (thanks @wookasz)
```
Comment (Kilo59) [inline docs/changelog.md]:
```suggestion
* [MAINTENANCE] Add docstrings for Validator and its save_expectation_suite and validate methods ([#6885](GxURL.PULL_REQUESTS/6885))
```
remove?

File: docs/changelog.md [hunk 3]
```diff
@@ -2,6 +2,79 @@
...
+* [MAINTENANCE] Clean up public_api excludes ([#6846](GxURL.PULL_REQUESTS/6846))
+* [MAINTENANCE] Fix the error message for invalid batch request options ([#6848](GxURL.PULL_REQUESTS/6848))
+* [MAINTENANCE] Standardize Store CRUD ([#6826](GxURL.PULL_REQUESTS/6826))
+* [MAINTENANCE] Fix scripts not found error in invoke ([#6867](GxURL.PULL_REQUESTS/6867))
+* [MAINTENANCE] Fix argument name typo ([#6850](GxURL.PULL_REQUESTS/6850)) (thanks @KirillUlich)
+* [MAINTENANCE] more clearly specifies range of supported python versions ([#6870](GxURL.PULL_REQUESTS/6870))
+* [MAINTENANCE] add validate_configuration docstring ([#6857](GxURL.PULL_REQUESTS/6857))
+* [MAINTENANCE] docstring for expect_column_values_to_not_be_null#validate_configuration ([#6859](GxURL.PULL_REQUESTS/6859))
+* [MAINTENANCE] Standardize project config CRUD ([#6837](GxURL.PULL_REQUESTS/6837))
+* [MAINTENANCE] update docstring in validator.py and checkpoint_result.py ([#6875](GxURL.PULL_REQUESTS/6875)) (thanks @allensallinger)
```
Comment (Kilo59) [inline docs/changelog.md]:
```suggestion
* [MAINTENANCE] update docstring in validator.py and checkpoint_result.py ([#6875](GxURL.PULL_REQUESTS/6875))
```
remove?

File: docs_rtd/changelog.rst [hunk 4]
```diff
@@ -4,6 +4,80 @@
...
+* [DOCS] enable running `invoke docstrings` on select modules ([#6868](GxURL.PULL_REQUESTS/6868))
+* [DOCS] Adds docstring for expect_column_distinct_values_to_contain_set ([#6855](GxURL.PULL_REQUESTS/6855))
+* [DOCS] Documentation Strings for Metric Domain Types and Metric Function Types ([#6872](GxURL.PULL_REQUESTS/6872))
+* [DOCS] added docstrings for the public API ([#6884](GxURL.PULL_REQUESTS/6884)) (thanks @sujensen)
+* [DOCS] Add public API docstring for `expect_column_values_to_be_unique` `validate_configuration` ([#6897](GxURL.PULL_REQUESTS/6897))
+* [DOCS] Miscellaneous docstrings for `DataContext` and utils ([#6852](GxURL.PULL_REQUESTS/6852))
+* [DOCS] Add public API docstring for `expect_column_values_to_be_of_type` `validate_configuration` ([#6896](GxURL.PULL_REQUESTS/6896))
+* [DOCS] Add public API docstring for `JsonSchemaProflier.validate` ([#6900](GxURL.PULL_REQUESTS/6900))
+* [DOCS] Exclude DataAssistantRunner.run() ([#6919](GxURL.PULL_REQUESTS/6919))
+* [DOCS] StoreValidationResultAction, StoreEvaluationParametersAction and StoreMetricsAction api docs ([#6879](GxURL.PULL_REQUESTS/6879)) (thanks @josectobar)
```
Comment (Kilo59) [inline docs_rtd/changelog.rst:30]:
```suggestion
* [DOCS] StoreValidationResultAction, StoreEvaluationParametersAction and StoreMetricsAction api docs ([#6879](GxURL.PULL_REQUESTS/6879))
```

File: docs_rtd/changelog.rst [hunk 5]
```diff
@@ -4,6 +4,80 @@
...
+* [MAINTENANCE] Clean up public_api excludes ([#6846](GxURL.PULL_REQUESTS/6846))
+* [MAINTENANCE] Fix the error message for invalid batch request options ([#6848](GxURL.PULL_REQUESTS/6848))
+* [MAINTENANCE] Standardize Store CRUD ([#6826](GxURL.PULL_REQUESTS/6826))
+* [MAINTENANCE] Fix scripts not found error in invoke ([#6867](GxURL.PULL_REQUESTS/6867))
+* [MAINTENANCE] Fix argument name typo ([#6850](GxURL.PULL_REQUESTS/6850)) (thanks @KirillUlich)
+* [MAINTENANCE] more clearly specifies range of supported python versions ([#6870](GxURL.PULL_REQUESTS/6870))
+* [MAINTENANCE] add validate_configuration docstring ([#6857](GxURL.PULL_REQUESTS/6857))
+* [MAINTENANCE] docstring for expect_column_values_to_not_be_null#validate_configuration ([#6859](GxURL.PULL_REQUESTS/6859))
+* [MAINTENANCE] Standardize project config CRUD ([#6837](GxURL.PULL_REQUESTS/6837))
+* [MAINTENANCE] update docstring in validator.py and checkpoint_result.py ([#6875](GxURL.PULL_REQUESTS/6875)) (thanks @allensallinger)
```
Comment (Kilo59) [inline docs_rtd/changelog.rst:64]:
```suggestion
* [MAINTENANCE] update docstring in validator.py and checkpoint_result.py ([#6875](GxURL.PULL_REQUESTS/6875))
```

File: docs_rtd/changelog.rst [hunk 6]
```diff
@@ -4,6 +4,80 @@
...
+* [MAINTENANCE] Use docusaurus style code block in api docs ([#6906](GxURL.PULL_REQUESTS/6906))
+* [MAINTENANCE] metrics linting ([#6889](GxURL.PULL_REQUESTS/6889))
+* [MAINTENANCE] Add exception message to `RenderedAtomicContent` failure renderer ([#6795](GxURL.PULL_REQUESTS/6795))
+* [MAINTENANCE] Remove CloudNotificationAction ([#6881](GxURL.PULL_REQUESTS/6881))
+* [MAINTENANCE] Use ruff linter for docstring linting ([#6913](GxURL.PULL_REQUESTS/6913))
+* [MAINTENANCE] Add validate_configuration method docstrings ([#6899](GxURL.PULL_REQUESTS/6899)) (thanks @elenajdanova)
+* [MAINTENANCE] docstring for expect_column_values_to_not_match_like_pattern_list's validate_configuration ([#6874](GxURL.PULL_REQUESTS/6874))
+* [MAINTENANCE] docstring for expect_column_values_to_not_match_like_pattern validate_configuration ([#6876](GxURL.PULL_REQUESTS/6876))
+* [MAINTENANCE] docstring for expect_compound_columns_to_be_unique validate_configuration ([#6878](GxURL.PULL_REQUESTS/6878))
+* [MAINTENANCE] Add docstrings for Validator and its save_expectation_suite and validate methods ([#6885](GxURL.PULL_REQUESTS/6885)) (thanks @wookasz)
```
Comment (Kilo59) [inline docs_rtd/changelog.rst:78]:
```suggestion
* [MAINTENANCE] Add docstrings for Validator and its save_expectation_suite and validate methods ([#6885](GxURL.PULL_REQUESTS/6885))
```

**Review by alexsherstinsky — APPROVED** [review]
LGTM!

File: docs/changelog.md (hunk 1 above)
Comment (NathanFarmer) [reply docs/changelog.md]: Yes please remove this and add josectobar to the `teams.yml`.

File: docs/changelog.md (hunk 3 above)
Comment (NathanFarmer) [reply docs/changelog.md]: Yes please remove this and add allensallinger to the `teams.yml`.

File: docs/changelog.md [hunk 7]
```diff
@@ -2,6 +2,79 @@
...
+* [MAINTENANCE] updated docstring on validate configuration ([#6871](GxURL.PULL_REQUESTS/6871))
+* [MAINTENANCE] Exclude unit tests from `comprehensive` stage of `dev` CI ([#6903](GxURL.PULL_REQUESTS/6903))
+* [MAINTENANCE] Refactor `file_relative_path` util ([#6778](GxURL.PULL_REQUESTS/6778))
+* [MAINTENANCE] switch to `ruff` linter ([#6888](GxURL.PULL_REQUESTS/6888))
+* [MAINTENANCE] Use docusaurus style code block in api docs ([#6906](GxURL.PULL_REQUESTS/6906))
+* [MAINTENANCE] metrics linting ([#6889](GxURL.PULL_REQUESTS/6889))
+* [MAINTENANCE] Add exception message to `RenderedAtomicContent` failure renderer ([#6795](GxURL.PULL_REQUESTS/6795))
+* [MAINTENANCE] Remove CloudNotificationAction ([#6881](GxURL.PULL_REQUESTS/6881))
+* [MAINTENANCE] Use ruff linter for docstring linting ([#6913](GxURL.PULL_REQUESTS/6913))
+* [MAINTENANCE] Add validate_configuration method docstrings ([#6899](GxURL.PULL_REQUESTS/6899)) (thanks @elenajdanova)
```
Comment (NathanFarmer) [inline docs/changelog.md]: Please remove this and add elenajdanova to the `teams.yml`.

File: docs/changelog.md (hunk 2 above)
Comment (NathanFarmer) [reply docs/changelog.md]: Please remove this and add wookasz to the `teams.yml`.

File: docs/changelog.md [hunk 8]
```diff
@@ -2,6 +2,79 @@
 title: Changelog
 ---
 
+### 0.15.45
+* [FEATURE] Experimental datasources `batch.head()` ([#6765](GxURL.PULL_REQUESTS/6765))
+* [FEATURE] Add Validation Result URL to Checkpoint Result ([#6908](GxURL.PULL_REQUESTS/6908)) (thanks @josectobar)
```
Comment (NathanFarmer) [inline docs/changelog.md]: Please remove this and add josectobar to the `teams.yml`.

File: docs/changelog.md [hunk 9]
```diff
@@ -2,6 +2,79 @@
...
+* [BUGFIX] Capital one contrib/micdavis/import hotfix ([#6922](GxURL.PULL_REQUESTS/6922)) (thanks @micdavis)
+* [DOCS] Adding docstring for `Checkpoint.self_check()` ([#6841](GxURL.PULL_REQUESTS/6841))
+* [DOCS] `AbstractDataContext.add_store` docstring ([#6851](GxURL.PULL_REQUESTS/6851))
+* [DOCS] Doc Strings for ExpectationSuite Display Methods ([#6856](GxURL.PULL_REQUESTS/6856))
+* [DOCS] `DataAssistantResult.get_expectation_suite()` docstring ([#6862](GxURL.PULL_REQUESTS/6862))
+* [DOCS] Misc docstrings around `DataAssistant` ([#6866](GxURL.PULL_REQUESTS/6866))
+* [DOCS] enable running `invoke docstrings` on select modules ([#6868](GxURL.PULL_REQUESTS/6868))
+* [DOCS] Adds docstring for expect_column_distinct_values_to_contain_set ([#6855](GxURL.PULL_REQUESTS/6855))
+* [DOCS] Documentation Strings for Metric Domain Types and Metric Function Types ([#6872](GxURL.PULL_REQUESTS/6872))
+* [DOCS] added docstrings for the public API ([#6884](GxURL.PULL_REQUESTS/6884)) (thanks @sujensen)
```
Comment (NathanFarmer) [inline docs/changelog.md]: Please remove this and add sujensen to the `teams.yml`.

File: docs/changelog.md [hunk 10]
```diff
@@ -2,6 +2,79 @@
...
+* [DOCS] Miscellaneous docstrings for `DataContext` and utils ([#6852](GxURL.PULL_REQUESTS/6852))
+* [DOCS] Add public API docstring for `expect_column_values_to_be_of_type` `validate_configuration` ([#6896](GxURL.PULL_REQUESTS/6896))
+* [DOCS] Add public API docstring for `JsonSchemaProflier.validate` ([#6900](GxURL.PULL_REQUESTS/6900))
+* [DOCS] Exclude DataAssistantRunner.run() ([#6919](GxURL.PULL_REQUESTS/6919))
+* [DOCS] StoreValidationResultAction, StoreEvaluationParametersAction and StoreMetricsAction api docs ([#6879](GxURL.PULL_REQUESTS/6879)) (thanks @josectobar)
+* [DOCS] Add public API docstring for `expect_column_values_to_be_dateutil_parseable` `validate_configuration` ([#6864](GxURL.PULL_REQUESTS/6864))
+* [DOCS] YAML docs ([#6861](GxURL.PULL_REQUESTS/6861))
+* [DOCS] Add public API docstring for `expect_column_values_to_be_decreasing` `validate_configuration` ([#6865](GxURL.PULL_REQUESTS/6865))
+* [DOCS] Docstrings for `Checkpoint` and related classes ([#6882](GxURL.PULL_REQUESTS/6882))
+* [DOCS] Add public API docstring for expect_table_row_count_to_be_between validate_configuration ([#6883](GxURL.PULL_REQUESTS/6883)) (thanks @lockettks)
```
Comment (NathanFarmer) [inline docs/changelog.md]: Please remove this and add lockettks to the `teams.yml`.

File: docs_rtd/changelog.rst [hunk 11]
```diff
@@ -4,6 +4,80 @@
 Changelog
 #########
 
+0.15.45
+-----------------
+* [FEATURE] Experimental datasources `batch.head()` ([#6765](GxURL.PULL_REQUESTS/6765))
```
Comment (NathanFarmer) [inline docs_rtd/changelog.rst]: Same comments throughout

**Comment by Kilo59** [comment]
Some issues to be worked out.
Do not merge

**Review by NathanFarmer — DISMISSED** [review]
Also, what's going on with the links? Something about an `Enum` maybe?

File: docs/changelog.md [hunk 12]
```diff
@@ -2,6 +2,79 @@
 title: Changelog
 ---
 
+### 0.15.45
+* [FEATURE] Experimental datasources `batch.head()` ([#6765](https://github.com/great-expectations/great_expectations/pull/6765))
+* [FEATURE] Add Validation Result URL to Checkpoint Result ([#6908](https://github.com/great-expectations/great_expectations/pull/6908)) (thanks @josectobar)
```
Comment (Kilo59) [inline docs/changelog.md]:
```suggestion
* [FEATURE] Add Validation Result URL to Checkpoint Result ([#6908](https://github.com/great-expectations/great_expectations/pull/6908))
```

File: docs/changelog.md [hunk 13]
```diff
@@ -2,6 +2,79 @@
...
+* [DOCS] `AbstractDataContext.add_store` docstring ([#6851](https://github.com/great-expectations/great_expectations/pull/6851))
+* [DOCS] Doc Strings for ExpectationSuite Display Methods ([#6856](https://github.com/great-expectations/great_expectations/pull/6856))
+* [DOCS] `DataAssistantResult.get_expectation_suite()` docstring ([#6862](https://github.com/great-expectations/great_expectations/pull/6862))
+* [DOCS] Misc docstrings around `DataAssistant` ([#6866](https://github.com/great-expectations/great_expectations/pull/6866))
+* [DOCS] enable running `invoke docstrings` on select modules ([#6868](https://github.com/great-expectations/great_expectations/pull/6868))
+* [DOCS] Adds docstring for expect_column_distinct_values_to_contain_set ([#6855](https://github.com/great-expectations/great_expectations/pull/6855))
+* [DOCS] Documentation Strings for Metric Domain Types and Metric Function Types ([#6872](https://github.com/great-expectations/great_expectations/pull/6872))
+* [DOCS] added docstrings for the public API ([#6884](https://github.com/great-expectations/great_expectations/pull/6884)) (thanks @sujensen)
```
Comment (Kilo59) [inline docs/changelog.md]:
```suggestion
* [DOCS] added docstrings for the public API ([#6884](https://github.com/great-expectations/great_expectations/pull/6884))
```

File: docs/changelog.md [hunk 14]
```diff
@@ -2,6 +2,79 @@
...
+* [MAINTENANCE] Standardize Store CRUD ([#6826](https://github.com/great-expectations/great_expectations/pull/6826))
+* [MAINTENANCE] Fix scripts not found error in invoke ([#6867](https://github.com/great-expectations/great_expectations/pull/6867))
+* [MAINTENANCE] Fix argument name typo ([#6850](https://github.com/great-expectations/great_expectations/pull/6850)) (thanks @KirillUlich)
+* [MAINTENANCE] more clearly specifies range of supported python versions ([#6870](https://github.com/great-expectations/great_expectations/pull/6870))
+* [MAINTENANCE] add validate_configuration docstring ([#6857](https://github.com/great-expectations/great_expectations/pull/6857))
+* [MAINTENANCE] docstring for expect_column_values_to_not_be_null#validate_configuration ([#6859](https://github.com/great-expectations/great_expectations/pull/6859))
+* [MAINTENANCE] Standardize project config CRUD ([#6837](https://github.com/great-expectations/great_expectations/pull/6837))
+* [MAINTENANCE] update docstring in validator.py and checkpoint_result.py ([#6875](https://github.com/great-expectations/great_expectations/pull/6875)) (thanks @allensallinger)
```
Comment (Kilo59) [inline docs/changelog.md]:
```suggestion
* [MAINTENANCE] update docstring in validator.py and checkpoint_result.py ([#6875](https://github.com/great-expectations/great_expectations/pull/6875))
```

File: docs/changelog.md [hunk 15]
```diff
@@ -2,6 +2,79 @@
...
+* [MAINTENANCE] Remove CloudNotificationAction ([#6881](https://github.com/great-expectations/great_expectations/pull/6881))
+* [MAINTENANCE] Use ruff linter for docstring linting ([#6913](https://github.com/great-expectations/great_expectations/pull/6913))
+* [MAINTENANCE] Add validate_configuration method docstrings ([#6899](https://github.com/great-expectations/great_expectations/pull/6899)) (thanks @elenajdanova)
+* [MAINTENANCE] docstring for expect_column_values_to_not_match_like_pattern_list's validate_configuration ([#6874](https://github.com/great-expectations/great_expectations/pull/6874))
+* [MAINTENANCE] docstring for expect_column_values_to_not_match_like_pattern validate_configuration ([#6876](https://github.com/great-expectations/great_expectations/pull/6876))
+* [MAINTENANCE] docstring for expect_compound_columns_to_be_unique validate_configuration ([#6878](https://github.com/great-expectations/great_expectations/pull/6878))
+* [MAINTENANCE] Add docstrings for Validator and its save_expectation_suite and validate methods ([#6885](https://github.com/great-expectations/great_expectations/pull/6885)) (thanks @wookasz)
```
Comment (Kilo59) [inline docs/changelog.md]:
```suggestion
* [MAINTENANCE] Add docstrings for Validator and its save_expectation_suite and validate methods ([#6885](https://github.com/great-expectations/great_expectations/pull/6885))
```

File: docs_rtd/changelog.rst [hunk 16]
```diff
@@ -4,6 +4,80 @@
 Changelog
 #########
 
+0.15.45
+-----------------
+* [FEATURE] Experimental datasources `batch.head()` ([#6765](https://github.com/great-expectations/great_expectations/pull/6765))
+* [FEATURE] Add Validation Result URL to Checkpoint Result ([#6908](https://github.com/great-expectations/great_expectations/pull/6908)) (thanks @josectobar)
```
Comment (Kilo59) [inline docs_rtd/changelog.rst]:
```suggestion
* [FEATURE] Add Validation Result URL to Checkpoint Result ([#6908](https://github.com/great-expectations/great_expectations/pull/6908))
```

File: docs_rtd/changelog.rst [hunk 17]
```diff
@@ -4,6 +4,80 @@
...
+* [MAINTENANCE] Standardize Store CRUD ([#6826](https://github.com/great-expectations/great_expectations/pull/6826))
+* [MAINTENANCE] Fix scripts not found error in invoke ([#6867](https://github.com/great-expectations/great_expectations/pull/6867))
+* [MAINTENANCE] Fix argument name typo ([#6850](https://github.com/great-expectations/great_expectations/pull/6850)) (thanks @KirillUlich)
+* [MAINTENANCE] more clearly specifies range of supported python versions ([#6870](https://github.com/great-expectations/great_expectations/pull/6870))
+* [MAINTENANCE] add validate_configuration docstring ([#6857](https://github.com/great-expectations/great_expectations/pull/6857))
+* [MAINTENANCE] docstring for expect_column_values_to_not_be_null#validate_configuration ([#6859](https://github.com/great-expectations/great_expectations/pull/6859))
+* [MAINTENANCE] Standardize project config CRUD ([#6837](https://github.com/great-expectations/great_expectations/pull/6837))
+* [MAINTENANCE] update docstring in validator.py and checkpoint_result.py ([#6875](https://github.com/great-expectations/great_expectations/pull/6875)) (thanks @allensallinger)
```
Comment (Kilo59) [inline docs_rtd/changelog.rst]:
```suggestion
* [MAINTENANCE] update docstring in validator.py and checkpoint_result.py ([#6875](https://github.com/great-expectations/great_expectations/pull/6875))
```

File: docs_rtd/changelog.rst [hunk 18]
```diff
@@ -4,6 +4,80 @@
...
+* [MAINTENANCE] Remove CloudNotificationAction ([#6881](https://github.com/great-expectations/great_expectations/pull/6881))
+* [MAINTENANCE] Use ruff linter for docstring linting ([#6913](https://github.com/great-expectations/great_expectations/pull/6913))
+* [MAINTENANCE] Add validate_configuration method docstrings ([#6899](https://github.com/great-expectations/great_expectations/pull/6899)) (thanks @elenajdanova)
+* [MAINTENANCE] docstring for expect_column_values_to_not_match_like_pattern_list's validate_configuration ([#6874](https://github.com/great-expectations/great_expectations/pull/6874))
+* [MAINTENANCE] docstring for expect_column_values_to_not_match_like_pattern validate_configuration ([#6876](https://github.com/great-expectations/great_expectations/pull/6876))
+* [MAINTENANCE] docstring for expect_compound_columns_to_be_unique validate_configuration ([#6878](https://github.com/great-expectations/great_expectations/pull/6878))
+* [MAINTENANCE] Add docstrings for Validator and its save_expectation_suite and validate methods ([#6885](https://github.com/great-expectations/great_expectations/pull/6885)) (thanks @wookasz)
```
Comment (Kilo59) [inline docs_rtd/changelog.rst]:
```suggestion
* [MAINTENANCE] Add docstrings for Validator and its save_expectation_suite and validate methods ([#6885](https://github.com/great-expectations/great_expectations/pull/6885))
```

File: docs_rtd/changelog.rst [hunk 19]
```diff
@@ -4,6 +4,80 @@
...
+* [MAINTENANCE] Refactor `file_relative_path` util ([#6778](https://github.com/great-expectations/great_expectations/pull/6778))
+* [MAINTENANCE] switch to `ruff` linter ([#6888](https://github.com/great-expectations/great_expectations/pull/6888))
+* [MAINTENANCE] Use docusaurus style code block in api docs ([#6906](https://github.com/great-expectations/great_expectations/pull/6906))
+* [MAINTENANCE] metrics linting ([#6889](https://github.com/great-expectations/great_expectations/pull/6889))
+* [MAINTENANCE] Add exception message to `RenderedAtomicContent` failure renderer ([#6795](https://github.com/great-expectations/great_expectations/pull/6795))
+* [MAINTENANCE] Remove CloudNotificationAction ([#6881](https://github.com/great-expectations/great_expectations/pull/6881))
+* [MAINTENANCE] Use ruff linter for docstring linting ([#6913](https://github.com/great-expectations/great_expectations/pull/6913))
+* [MAINTENANCE] Add validate_configuration method docstrings ([#6899](https://github.com/great-expectations/great_expectations/pull/6899)) (thanks @elenajdanova)
```
Comment (Kilo59) [inline docs_rtd/changelog.rst]:
```suggestion
* [MAINTENANCE] Add validate_configuration method docstrings ([#6899](https://github.com/great-expectations/great_expectations/pull/6899))
```

File: docs/changelog.md [hunk 20]
```diff
@@ -2,6 +2,79 @@
...
+* [MAINTENANCE] Refactor `file_relative_path` util ([#6778](https://github.com/great-expectations/great_expectations/pull/6778))
+* [MAINTENANCE] switch to `ruff` linter ([#6888](https://github.com/great-expectations/great_expectations/pull/6888))
+* [MAINTENANCE] Use docusaurus style code block in api docs ([#6906](https://github.com/great-expectations/great_expectations/pull/6906))
+* [MAINTENANCE] metrics linting ([#6889](https://github.com/great-expectations/great_expectations/pull/6889))
+* [MAINTENANCE] Add exception message to `RenderedAtomicContent` failure renderer ([#6795](https://github.com/great-expectations/great_expectations/pull/6795))
+* [MAINTENANCE] Remove CloudNotificationAction ([#6881](https://github.com/great-expectations/great_expectations/pull/6881))
+* [MAINTENANCE] Use ruff linter for docstring linting ([#6913](https://github.com/great-expectations/great_expectations/pull/6913))
+* [MAINTENANCE] Add validate_configuration method docstrings ([#6899](https://github.com/great-expectations/great_expectations/pull/6899)) (thanks @elenajdanova)
```
Comment (Kilo59) [inline docs/changelog.md]:
```suggestion
* [MAINTENANCE] Add validate_configuration method docstrings ([#6899](https://github.com/great-expectations/great_expectations/pull/6899))
```

File: docs/changelog.md [hunk 21]
```diff
@@ -2,6 +2,79 @@
...
+* [DOCS] added docstrings for the public API ([#6884](https://github.com/great-expectations/great_expectations/pull/6884))
+* [DOCS] Add public API docstring for `expect_column_values_to_be_unique` `validate_configuration` ([#6897](https://github.com/great-expectations/great_expectations/pull/6897))
+* [DOCS] Miscellaneous docstrings for `DataContext` and utils ([#6852](https://github.com/great-expectations/great_expectations/pull/6852))
+* [DOCS] Add public API docstring for `expect_column_values_to_be_of_type` `validate_configuration` ([#6896](https://github.com/great-expectations/great_expectations/pull/6896))
+* [DOCS] Add public API docstring for `JsonSchemaProflier.validate` ([#6900](https://github.com/great-expectations/great_expectations/pull/6900))
+* [DOCS] Exclude DataAssistantRunner.run() ([#6919](https://github.com/great-expectations/great_expectations/pull/6919))
+* [DOCS] StoreValidationResultAction, StoreEvaluationParametersAction and StoreMetricsAction api docs ([#6879](https://github.com/great-expectations/great_expectations/pull/6879)) (thanks @josectobar)
```
Comment (cdkini) [inline docs/changelog.md]: Looks like Jose's attribution is still here

Comment (Kilo59) [reply docs/changelog.md]: Good catch. Thanks

Comment (Kilo59) [inline docs/changelog.md]:
```suggestion
* [DOCS] StoreValidationResultAction, StoreEvaluationParametersAction and StoreMetricsAction api docs ([#6879](https://github.com/great-expectations/great_expectations/pull/6879))
```

File: docs_rtd/changelog.rst [hunk 22]
```diff
@@ -4,6 +4,80 @@
...
+* [DOCS] added docstrings for the public API ([#6884](https://github.com/great-expectations/great_expectations/pull/6884)) (thanks @sujensen)
+* [DOCS] Add public API docstring for `expect_column_values_to_be_unique` `validate_configuration` ([#6897](https://github.com/great-expectations/great_expectations/pull/6897))
+* [DOCS] Miscellaneous docstrings for `DataContext` and utils ([#6852](https://github.com/great-expectations/great_expectations/pull/6852))
+* [DOCS] Add public API docstring for `expect_column_values_to_be_of_type` `validate_configuration` ([#6896](https://github.com/great-expectations/great_expectations/pull/6896))
+* [DOCS] Add public API docstring for `JsonSchemaProflier.validate` ([#6900](https://github.com/great-expectations/great_expectations/pull/6900))
+* [DOCS] Exclude DataAssistantRunner.run() ([#6919](https://github.com/great-expectations/great_expectations/pull/6919))
+* [DOCS] StoreValidationResultAction, StoreEvaluationParametersAction and StoreMetricsAction api docs ([#6879](https://github.com/great-expectations/great_expectations/pull/6879)) (thanks @josectobar)
```
Comment (Kilo59) [inline docs_rtd/changelog.rst]:
```suggestion
* [DOCS] StoreValidationResultAction, StoreEvaluationParametersAction and StoreMetricsAction api docs ([#6879](https://github.com/great-expectations/great_expectations/pull/6879))
```

**Pull request merged**